src/Stripe.net/Entities/Accounts/Account.cs

```diff
@@ -7,67 +7,148 @@ namespace Stripe
 
     public class Account : StripeEntity<Account>, IHasId, IHasMetadata, IHasObject, IPaymentSource
     {
+        /// <summary>
+        /// Unique identifier for the object.
+        /// </summary>
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        /// <summary>
+        /// String representing the object's type. Objects of the same type share the same value.
+        /// </summary>
         [JsonProperty("object")]
         public string Object { get; set; }
 
+        /// <summary>
+        /// Business information about the account.
+        /// </summary>
         [JsonProperty("business_profile")]
         public AccountBusinessProfile BusinessProfile { get; set; }
 
+        /// <summary>
+        /// The business type.
+        /// Possible values:
+        ///  <c>company</c>
+        ///  <c>government_entity</c>
+        ///  <c>individual</c>
+        ///  <c>non_profit</c>
+        /// </summary>
         [JsonProperty("business_type")]
         public string BusinessType { get; set; }
 
+        /// <summary>
+        /// TODO: Field description missing from spec.
+        /// </summary>
```

Comment:
Let's not commit this but just keep this as a solution in your docs generation to easily go and track those down